### DISCUSSION TITLE
ScrollBar background color

### DISCUSSION THREAD

**Discussion by andreymarchenko** [microsoft/monaco-editor#3752] [Q&A]
Hello, is it possible to set scroll background color different from editor background color?
https://github.com/microsoft/monaco-editor/issues/908 helps, but minimap background color also change.

**Comment by postatum** [comment]
I got the same issue. Even though the colour settings field/group is called `scrollbarSlider`, it actually changes a minimap colours.

**Comment by ghost** [comment]
+1
